docs/literate/src/files/getting_started_with_Trixi.jl

```diff
@@ -0,0 +1,287 @@
+#src # Getting started with Trixi.jl
+
+# This tutorial is intended for beginners in Julia and Trixi.jl.
+# After reading it, you will install Julia and Trixi on your PC and will be able to download files
+# from Trixi github, execute and edit them.
+# **Trixi.jl** is a numerical simulation framework for hyperbolic conservation laws 
+# written in [`Julia`](https://julialang.org/).
+# This means that Julia have to be installed on a PC to work with Trixi. 
+
+# ## Julia installation
+
+# Trixi works with the current stable Julia release. More information about Julia support can be
```

Comment:
```suggestion
# Trixi.jl works with the current stable Julia release. More information about Julia support can be
```
Please always use "Trixi.jl" (i.e., with the ".jl") when referring to Trixi.jl. Please fix this also in all other locations where this might be used wrongly in this PR.